_i18n/de/resources/moneropedia/node.md

```diff
@@ -1,24 +1,24 @@
 ---
-terms: ["node", "nodes", "full-node", "full-nodes"]
-summary: "a device on the Internet running the Monero software, with a full copy of the Monero blockchain, actively assisting the Monero network"
+terms: ["node", "nodes", "full-node", "full-nodes", "Node", "Nodes", "Full-Node", "Full-Nodes"]
+summary: "Ein mit dem Internet verbundenes Gerät mit einer Vollkopie der Monero-Blockchain, welches die Monero-Software ausführt und das Monero-Netzwerk aktiv unterstützt"
```

Comment:
```suggestion
summary: "Ein mit dem Internet verbundenes Gerät, welches die Monero-Software ausführt, über eine vollständige Kopie der Monero-Blockchain verfügt, und damit das Monero-Netzwerk aktiv unterstützt"
```

"Vollkopie" isn't commonly used (if at all? I have never heard it before). Changed the order so "software" comes first, followed by the "copy of the blockchain" - like in the English version. This somehow reads better because you also run the software first, which then downloads a copy of the blockchain. Feel free to revert, but I'd keep "vollständige Kopie" instead of "Vollkopie".

Reply:
Much better like that, thanks!